src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.cs

```diff
@@ -82,17 +80,15 @@ public async Task<NavigateToSearchLocation> SearchDocumentAsync(Document documen
                 document, searchPattern, kinds, onItemFound, cancellationToken).ConfigureAwait(false);
         }
 
-        public async Task<NavigateToSearchLocation> SearchProjectAsync(Project project, ImmutableArray<Document> priorityDocuments, string searchPattern, IImmutableSet<string> kinds, Func<INavigateToSearchResult, Task> onResultFound, bool isFullyLoaded, CancellationToken cancellationToken)
+        public async Task SearchProjectAsync(Project project, ImmutableArray<Document> priorityDocuments, string searchPattern, IImmutableSet<string> kinds, bool isFullyLoaded, Func<INavigateToSearchResult, Task> onResultFound, CancellationToken cancellationToken)
```

Comment:
```suggestion
        public async Task SearchProjectAsync(
        	Project project,
        	ImmutableArray<Document> priorityDocuments,
        	string searchPattern,
        	IImmutableSet<string> kinds,
        	bool isFullyLoaded,
        	Func<INavigateToSearchResult, Task> onResultFound,
        	CancellationToken cancellationToken)
```

Reply:
and other places...

Reply:
going to rollt his into the followup PR.